tox.ini

```diff
@@ -19,7 +19,7 @@ commands =
     # -l: show locals in tracebacks
     # --tb=short: short traceback print mode
     # --strict: marks not registered in configuration file raise errors
-    pytest --cov=code42cli --cov-report xml -v -rsxX -l --tb=short --strict -m "not integration"
+    pytest --cov=code42cli --cov-report xml -v -rsxX -l --tb=short --strict --ignore=integration
```

Comment:
This is the benefit of _still_ keeping the integration tests in their own directory... Ignoring a whole directory is significantly faster than ignoring individual tests with a `pytest mark`.

Reply:
This is nice, as I marked the individual test class as `integration`, some of the negative tests which were kind of unit tests still exists in `integration` directory, we might want to move them back to appropriate unit test files.

I can create a pull request against this branch with the changes and then we can merge this to master to make sure all unit tests are executed.